openmc/mgxs/mgxs.py

```diff
@@ -1547,7 +1546,7 @@ def tallies(self):
         if self._tallies is None:
 
             # Create a list of scores for each Tally to be created
-            scores = ['flux', 'total', 'scatter-P1']
+            scores = ['flux', 'total', 'scatter-1']
```

Comment:
Good catch, the `scatter-0` order that was brought in by `scatter-P1` was wasteful.
However, should this be `scatter-1` or `nu-scatter-1`?